discord_presence.c

```diff
@@ -231,6 +231,15 @@ discord_presence_message (uint32_t id, uintptr_t ctx, uint32_t p1, uint32_t p2)
                 }
             }
             break;
+        case DB_EV_SONGFINISHED:
+            if(discord_enabled) {
```

Comment:
You should usually follow code style as it originally is. I got used to [deadbeef code style](https://github.com/DeaDBeeF-Player/deadbeef/wiki/Code-guidelines) so that's how I write by default :)  A space after if would be needed here.